src/test/java/fr/lunatech/timekeeper/resources/utils/InternalResourceUtils.java

```diff
@@ -163,24 +164,22 @@ static ValidatableResponse readResourceValidation(String uri_root, String token)
     /**
      * Allows the caller to specify as parameter the statusCode
      *
-     * @param m      HttpMethod
      * @param uri    uri
      * @param token  security token
-     * @return
+     * @return a ValidatableResponse
      */
-    static ValidatableResponse resourceValidation(VerbDefinition m, String uri, String token) {
-        return resourceValidation(m, uri, token, Option.none());
+    static ValidatableResponse resourceValidation(String uri, String token) {
```

Comment:
Imho, we could remove this and reuse the 

static <T> ValidatableResponse resourceValidation(VerbDefinition m, String uri, String token, T body) {
        return resourceValidation(m, uri, token, Option.some(body));

and pass the VERB = GET in ResourceValidation.class -> line 37

Reply:
Yes. This is a refactor related to SonarCloud but I am not sure it is correct. I might revert and switch back to the old version of resourceValidation method.